metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/parser/TypeFromSet.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.parser;
+
+import java.util.List;
+
+public interface TypeFromSet<SUBSET, SUPERSET> {
```

Comment:
It is definitely not superset - subset relationship. It is more like a collection vs element to me.

Reply:
done